content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+Making use of the <<cli#,Jenkins CLI>> it is also possible to enable and disable
+plugins. Since Jenkins 2.136, administrators may invoke the `enable-plugin` 
+command to re-enable again a collection of plugins which were previously disabled
+and since Jenkins 2.151, the `disable-plugin` command is included, which provides
+a command to disable installed plugins.
+
+The `enable-plugin` command receives as parameter a list of plugins to be enabled.
```

Comment:
It seems that the common practice is just to explain the command (at least in the examples I found). Also, I don't think that an explanation of the internal behaviour would give real value. The help message should be self-explanatory so that the command can be clear enough on how to use it.